incubating.rst

```diff
@@ -24,11 +24,17 @@ This generator is designed as a replacement of the current ``CMakeDeps`` generat
 - Executables from ``requires`` can also be used in non cross-build scenarios. When a ``tool_requires`` to the same depependency exists, then those executables will have priority.
 - Creation of a new ``conan_cmakedeps_paths.cmake`` that contains definitions of ``<pkg>_DIR`` paths for direct finding of the dependencies. This file is also planned to be used in ``cmake-conan`` to extend its usage and avoid some current limitations due to the fact that a CMake driven installation cannot inject a toolchain later.
 
+.. note::
+   
+   This generator is only intended to generate ``config.cmake`` config files, it will not generate ``Find.cmake`` find modules, it is not planned to add support.
```

Comment:
```suggestion
   This generator is only intended to generate ``config.cmake`` config files, it will not generate ``Find*.cmake`` find modules, and support for it is not planned.
```